### PR TITLE
Update Get-CsTeamsAppPermissionPolicy.md

### PR DESCRIPTION
**Pull request by xuanzhou4**
Add some usage examples and parameters description for Get-CsTeamsAppPermissionPolicy.

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 1]
```diff
@@ -38,7 +38,45 @@ As an admin, you can use app permission policies to enable or block specific app
 ## EXAMPLES
 
 ### Example 1
-Intentionally omitted.
+
+Get the Global Teams App Permission Policy.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:42]:
```suggestion
Get the Global Teams App Permission Policy.

```

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 2]
```diff
@@ -38,7 +38,45 @@ As an admin, you can use app permission policies to enable or block specific app
...
+Get-CsTeamsAppPermissionPolicy -Identity Global
+
+Identity               : Global
+DefaultCatalogApps     : {Id=26bc2873-6023-480c-a11b-76b66605ce8c, Id=0d820ecd-def2-4297-adad-78056cde7c78, Id=com.microsoft.teamspace.tab.planner}
+GlobalCatalogApps      : {}
+PrivateCatalogApps     : {}
+Description            :
+DefaultCatalogAppsType : AllowedAppList
+GlobalCatalogAppsType  : AllowedAppList
+PrivateCatalogAppsType : AllowedAppList
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:53]: Please use the output format:
https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide#executable-examples

Thanks!

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 3]
```diff
@@ -38,7 +38,45 @@ As an admin, you can use app permission policies to enable or block specific app
...
+
+Identity               : Tag:test
+DefaultCatalogApps     : {Id=26bc2873-6023-480c-a11b-76b66605ce8c, Id=0d820ecd-def2-4297-adad-78056cde7c78, Id=com.microsoft.teamspace.tab.planner}
+GlobalCatalogApps      : {}
+PrivateCatalogApps     : {}
+Description            :
+DefaultCatalogAppsType : AllowedAppList
+GlobalCatalogAppsType  : AllowedAppList
+PrivateCatalogAppsType : AllowedAppList
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:78]: Same here.

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 4]
```diff
@@ -58,7 +96,7 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
-Do not use.
+Name of App setup permission policy.If empty, all Identities will be used by default.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:99]:
```suggestion
Name of App setup permission policy. If empty, all Identities will be used by default.
```

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md (hunk 1 above)
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:42]: Please move the description below the code

**Review by dariomws — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 5]
```diff
@@ -38,7 +38,49 @@ As an admin, you can use app permission policies to enable or block specific app
...
+Identity               : Global
+DefaultCatalogApps     : {Id=26bc2873-6023-480c-a11b-76b66605ce8c, Id=0d820ecd-def2-4297-adad-78056cde7c78, Id=com.microsoft.teamspace.tab.planner}
+GlobalCatalogApps      : {}
+PrivateCatalogApps     : {}
+Description            :
+DefaultCatalogAppsType : AllowedAppList
+GlobalCatalogAppsType  : AllowedAppList
+PrivateCatalogAppsType : AllowedAppList
+```
+Get the Global Teams App Permission Policy.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:56]:
```suggestion
Get the global Teams app permission policy.

```

File: skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md [hunk 6]
```diff
@@ -38,7 +38,49 @@ As an admin, you can use app permission policies to enable or block specific app
...
+Identity               : Tag:test
+DefaultCatalogApps     : {Id=26bc2873-6023-480c-a11b-76b66605ce8c, Id=0d820ecd-def2-4297-adad-78056cde7c78, Id=com.microsoft.teamspace.tab.planner}
+GlobalCatalogApps      : {}
+PrivateCatalogApps     : {}
+Description            :
+DefaultCatalogAppsType : AllowedAppList
+GlobalCatalogAppsType  : AllowedAppList
+PrivateCatalogAppsType : AllowedAppList
+```
+Get all the Teams App Permission Policies.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md:83]:
```suggestion
Get all the Teams app permission policies.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xuanzhou4, kindy commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 @dariomws

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Pull request merged**